x-pack/plugins/cases/public/containers/use_get_case_user_actions.test.tsx

```diff
@@ -18,7 +18,9 @@ import {
   basicPushSnake,
   caseUserActions,
   elasticUser,
+  getJiraConnectorWithoutId,
```

Comment:
This file's tests changed because the connector id needed to be extracted out to the `*_val_connector_id` fields. I also created some helper functions to describe which kind of user action is being created.